### PR TITLE
Add data input info

### PR DESCRIPTION
**Pull request by seb231**

### CODE REVIEW

File: doc/intro.md [hunk 1]
```diff
@@ -64,6 +64,33 @@ See below the cohort component method we aim to provide:
...
+Note:
+
+We will ultimately provide four alternatives for projecting the components of change (i.e. fertiliy, mortality and migration): projected values or rates can either be fixed, or determined by applying a national trend.
+See below:
+
+![CCM - projection alternatives](images/Component_projection_alternatives.png)
+
+For estimating the value or rate for the first year of projection (first projection year or base year), we will provide three alternatives. See below:
+
+![CCM - base year alternatives](images/First_projection_year_projection_alternatives.png)
```
Comment (sunnytownsend) [inline doc/intro.md:76]: change the image label to `CCM - first year projection alternatives`

Comment (seb231) [reply doc/intro.md:76]: thanks, should have done this in the last branch!

**Comment by seb231** [comment]
I just massively messed up the merging/history, but I think it's right now...

**Comment by sunnytownsend** [comment]
:shipit:

**Pull request merged**